### PR TITLE
METRICS-2: Add default openmrs columns

### PR DESCRIPTION
**Pull request by Akayeshmantha**
### Description of what I changed
Introduce default openmrs liquibase columns.

**Issue I worked on**
see :- https://issues.openmrs.org/browse/METRICS-2

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@Akayeshmantha  do you think we could make a ticket for this?

**Comment by Akayeshmantha** [comment]
@HerbertYiga  of course lets create issues here in the repo

**Comment by ibacher** [comment]
@HerbertYiga @Akayeshmantha Since we don't normally encourage GitHub issues, I've created a Jira project for this [here](https://issues.openmrs.org/projects/METRICS/summary). You both should have appropriate permissions, but let me know if there's anything wrong with the setup.

**Comment by Akayeshmantha** [comment]
@ibacher  fixed the review comments here.

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -15,12 +15,23 @@
             </column>
             <column name="uuid" type="varchar(40)" />
             <column name="title" type="varchar(255)" />
-            <column name="timeStamp" type="timestamp" defaultValueDate="CURRENT_TIMESTAMP"/>
+            <column name="timeStamp" type="DATETIME"/>
```
Comment (ibacher) [inline api/src/main/resources/liquibase.xml]: This field should be named `timestamp` or `time_stamp` for consistency with how we define database field names.

Comment (Akayeshmantha) [reply api/src/main/resources/liquibase.xml]: sure @ibacher fixed it.

**Comment by Akayeshmantha** [comment]
@ibacher  pushed the changes.

**Review by ibacher — APPROVED** [review]

**Pull request merged**